app/schemas/lt.vilnius.tvarkau.database.AppDatabase/1.json

```diff
@@ -0,0 +1,157 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 1,
+    "identityHash": "089f62ceebb7dee33d6bff3eab2ed8f6",
+    "entities": [
+      {
+        "tableName": "reports",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`id` INTEGER NOT NULL, `user_id` INTEGER NOT NULL, `ref_no` TEXT, `report_type_id` INTEGER NOT NULL, `lat` REAL NOT NULL, `lng` REAL NOT NULL, `status_status_id` INTEGER NOT NULL, `description` TEXT NOT NULL, `answer` TEXT, `license_plate_no` TEXT, `registered_at` TEXT, `completed_at` TEXT, PRIMARY KEY(`id`))",
+        "fields": [
+          {
+            "fieldPath": "id",
+            "columnName": "id",
+            "affinity": "INTEGER",
+            "notNull": true
+          },
+          {
+            "fieldPath": "userId",
+            "columnName": "user_id",
+            "affinity": "INTEGER",
+            "notNull": true
+          },
+          {
+            "fieldPath": "refNo",
+            "columnName": "ref_no",
+            "affinity": "TEXT",
+            "notNull": false
```

Comment:
How can it be nullable ?

Reply:
Now Avilys and backend works async. So initially `refNo` might be null and later, when document is registered in Avilys record will be updated.